src/tests/suite/daos_rebuild_ec.c

```diff
@@ -65,33 +66,19 @@ rebuild_ec_internal(void **state, uint16_t oclass, int kill_data_nr,
 		rebuild_pools_ranks(&arg, 1, &extra_kill_ranks[2], 2, false);
 	}
 
-	if (write_type == PARTIAL_UPDATE)
-		verify_ec_partial(&req, arg->index, 0);
-	else if (write_type == FULL_UPDATE)
-		verify_ec_full(&req, arg->index, 0);
-	else if (write_type == FULL_PARTIAL_UPDATE)
-		verify_ec_full_partial(&req, arg->index, 0);
-	else if (write_type == PARTIAL_FULL_UPDATE)
-		verify_ec_full(&req, arg->index, 0);
 	ioreq_fini(&req);
 
+	rc = daos_obj_verify(arg->coh, oid, DAOS_EPOCH_MAX);
+	assert_int_equal(rc, 0);
```

Comment:
daos_obj_verify only can check the data consistency (for example replica ext match), but cannot check if the data match with expected buffer right?
seems better to keep original verify_ec_xxx() check, you may also add the daos_obj_verify() here.

Reply:
indeed. I will add it back if it needs refresh.